ranger/core/fm.py

```diff
@@ -293,15 +293,15 @@ def copy(src, dest):
                     shutil.copy(self.relpath(src), self.confpath(dest))
                 except OSError as ex:
                     sys.stderr.write("  ERROR: %s\n" % str(ex))
-        if which == 'rifle' or which == 'all':
```

Comment:
I don't disagree with these changes, just seems a little unfortunate to tie them up with a fairly major change to plugins.